platform/darwin/test/MGLStyleTests.mm

```diff
@@ -470,4 +470,24 @@ - (void)testLanguageMatching {
     }
 }
 
+#pragma mark Transition tests
+
+- (void)testTransition
+{
+    MGLTransition transitionTest = MGLTransitionMake(5, 4);
+    
+    self.style.transition = transitionTest;
+    
+    XCTAssert(self.style.transition.delay == transitionTest.delay);
+    XCTAssert(self.style.transition.duration == transitionTest.duration);
```

Comment:
Does this also need to test `enablePlacementTransitions`?

Reply:
Since I split the functionality in `transition` and `enablePlacementTransitions` I'm testing the latter in it's own test method.

Reply:
Ah yes, I'm thinking of a previous discussion I saw on the previous PR!